File: codewars/csharp/7 _ Highest and Lowest.cs
//In this little assignment you are given a string of space separated numbers, and have to return the highest and lowest number.
//Kata.HighAndLow("1 2 3 4 5"); // return "5 1"
using System;
using System.Linq;

public static class Kata
{
  public static string HighAndLow(string numbers)
  {
    int[] numarray = numbers.Split(" ").Select(n => Convert.ToInt32(n)).ToArray();
    return numarray.Max()+" "+numarray.Min();
  }
}

File: codewars/python/7 _ Highest and Lowest.py
#In this little assignment you are given a string of space separated numbers, and have to return the highest and lowest number.
#high_and_low("1 2 -3 4 5") # return "5 -3"
high_and_low = lambda numbers : str(max((int(i) for i in numbers.split())))+" "+str(min((int(i) for i in numbers.split())))
